metrics.go

```diff
@@ -182,6 +182,21 @@ const (
 	// MetricBackendWriteFailedRequests measures failed backend write requests count
 	MetricBackendWriteFailedRequests = "backend_write_requests_failed_total"
 
+	// MetricBackendAtomicWriteRequests measures backend atomic write requests count
+	MetricBackendAtomicWriteRequests = "backend_atomic_write_requests_total"
+
+	// MetricBackendAtomicWriteFailedRequests measures failed backend atomic write requests count
+	MetricBackendAtomicWriteFailedRequests = "backend_atomic_write_requests_failed_total"
+
+	// MetricBackendAtomicWriteConditionFailed measures the amount of atomic write requests that result in condition failure.
+	MetricBackendAtomicWriteConditionFailed = "backend_atomic_write_condition_failed_total"
+
+	// MetricBackendAtomicWriteHistogram measures histogram of backend write latencies
+	MetricBackendAtomicWriteHistogram = "backend_atomic_write_seconds"
+
+	// MetricBackendAtomicWriteContention measures the amount of contention experienced by atomic write operations.
+	MetricBackendAtomicWriteContention = "backend_atomic_write_contention"
```

Comment:
This godoc was not clear to me whether we were tracking the amount of operations or the amount of some time measurement (in seconds?) of contention.
I think using the metric's `Help` message here, would further clarify.